README.md

```diff
@@ -1,42 +1,422 @@
-The local device is illustrated with white text.
-In the upper-right corner, the status of the communication link between each spacecraft is shown.
-Finally, the time in the lower left and lower right corners correspond to the epoch and the PASEOS local simulation time.
-
 <p align="center">
   <a href="https://github.com/aidotse/PASEOS/">
-    <img src="resources/images/animation.png" alt="Scheme"  width="900" height="500">
+    <img src="resources/images/flowchart.svg" alt="Scheme"  width="910" height="459">
   </a>
   <p align="center">
-    Snapshot of PASEOS visualization
+    Description of PASEOS workflow on an individual device
   </p>
-</p>
\ No newline at end of file
+</p>
+
+## Glossary
+* ### Activity
+  An activity is the abstraction that PASEOS uses to keep track of specific actions performed by a [SpacecraftActor](#spacecraftactor) upon a request from the user. To register an activity, an user shall first create an [activity function](#activity-function), which describes the operation to be performed, and provide information on the power-consumption due to the activity execution. <br>PASEOSis responsible for the execution of the activity and for updating the system status depending on the effects of the activity (e.g., by discharging they satellite battery).<br>
+  When registering an activity, the user can specify a [constraint-function](#constraint-function) to specify constraints to be met during the execution of the activity and an an [on termination](#on-termination) function to specify additional operations to performed by PASEOS on termination of the activity function.
+
+
+* ### Activity function
+  User-defined function emulating any operation to be executed in a PASEOS a [SpacecraftActor](#spacecraftactor). Activity functions are necessayr to register [activities](#activity). Activity functions might include data transmission, house-keeping operations, onboard data acquisition and processing, and others.
+
+* ### Actor
+  Since PASEOS is fully-decentralised, each node of a PASEOS constellation shall run an instance of PASEOS modelling all the nodes of  that constellation.  The abstraction of a constellation node inside a PASEOS instace is a PASEOS `actor`. 
+
+* ### Constraint-function
+  A constraint function is an asynchronous function that can be used by the PASEOS user to specify some constraints that shall be met during the execution of an activity.
+
+* ### Ground stationActor
+  `PASEOS actor` emulating a ground station.  
+
+* ### Local actor
+  The `local actor` in a PASEOS instance is the `actor` that models the behavior and the status of the node that runs that PASEOS instance.
+
+* ### On-termination function
+  A on-termination function is an asynchronous function that can be used by the PASEOS user to specify some operations to be executed on termination of predefied PASEOS user.
+
+* ### SpacecraftActor
+  `PASEOS actor` emulating a spacecraft or a satellite. 
```

Comment:
pls fit style here and in vicinity to match the rest (just PASEOS no `text`, and actor as link to glossary item)